airflow/config_templates/config.yml

```diff
@@ -1130,6 +1130,56 @@ metrics:
       type: string
       example: ~
       default: "False"
+traces:
+  description: |
+    Distributed traces integration settings.
+  options:
+    otel_on:
+      description: |
+        Enables sending traces to OpenTelemetry.
+      version_added: 2.9.0
+      type: string
+      example: ~
+      default: "False"
+    otel_host:
+      description: |
+        Specifies the hostname or IP address of the OpenTelemetry Collector to which Airflow sends
+        traces.
```

Comment:
Perhaps a new root-level `otel_integration_on` should control all otel features and each component can be overridden with its specific switch?

```
AIRFLOW__CORE__OTEL_INTEGRATION_ON = True
AIRFLOW__METRICS__OTEL_ON = False
```

would turn all otel features on, then turn off otel metrics (leaving traces, and whatever comes in the future on).  I guess that can come later if at all.

Reply:
That is a good topic. I kind of thought about combining all together, but then resorted to them being separate, because I know that some users would not want to have traces turned on, while sending metrics, or vice versa. The reason being is that users may not want traces for cost or simply they are not using it. In some other cases, users may want to send metrics to metrics monitoring system only, or directly (e.g. chronosphere), or have traces sent to some other traces system separately (e.g. Jaeger, zipkin, or honeycomb). I know we assume that users would definitely use OTEL collector to do that, but some users may prefer skipping the agent and just sending the OTEL metrics right into some end point directly.

For now, I'd like to have them separated , and this may be true for the OTEL logs as well.